File: TMessagesProj/src/main/java/tw/nekomimi/nekogram/cc/CCConverter.java
package tw.nekomimi.nekogram.cc;
public class CCConverter {
    public static final int PILLS = 0;
    public static final int PURE = 1;
    public static CCConverter get(CCTarget target) { return new CCConverter(); }
    public static String convert(String text, CCTarget target) { return text; }
    public static String convert(String text, int mode) { return text; }
    public String convert(String text) { return text; }
}

File: TMessagesProj/src/main/java/tw/nekomimi/nekogram/cc/CCConverter.kt
package tw.nekomimi.nekogram.cc

/**
 * Stub: Chinese character converter - returns input unchanged
 */
class CCConverter {
    companion object {
        @JvmStatic
        fun get(target: String): CCConverter {
            return CCConverter()
        }

        @JvmStatic
        fun get(target: CCTarget): CCConverter {
            return CCConverter()
        }
    }

    fun convert(text: String): String = text
}
